conan_test_azure/Dockerfile

```diff
@@ -42,19 +42,22 @@ RUN dpkg --add-architecture i386 \
        python3.8-dev \
        golang \
        pkg-config \
-       && rm -rf /var/lib/apt/lists/* \
-       && wget -q --no-check-certificate https://cmake.org/files/v${CMAKE_VERSION_MAJOR_MINOR}/cmake-${CMAKE_VERSION_FULL}-Linux-x86_64.tar.gz \
-       && tar -xzf cmake-${CMAKE_VERSION_FULL}-Linux-x86_64.tar.gz \
+       && rm -rf /var/lib/apt/lists/*
+
+RUN wget -q --no-check-certificate -O /tmp/cmake-${CMAKE_VERSION_FULL}-Linux-x86_64.tar.gz https://cmake.org/files/v${CMAKE_VERSION_MAJOR_MINOR}/cmake-${CMAKE_VERSION_FULL}-Linux-x86_64.tar.gz \
```

Comment:
These env vars are not defined for this image

Reply:
ouch!

Reply:
Sorry for having bothering you for such a stupid bug 🙈

Reply:
What?! You found the bug first hahah it's not your fault, we should found it at first commit (my fault)

Reply:
Yeah, but I introduced this bug in my PR 

Probably with one of my mass search-and-replaces